src/query-rule-context/query-rule-context.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, Inject, forwardRef } from '@angular/core';
+
+import { connectQueryRules } from 'instantsearch.js/es/connectors';
+import { BaseWidget } from '../base-widget';
+import { NgAisInstantSearch } from '../instantsearch/instantsearch';
+
+type FacetValue = string | number;
```

Comment:
can't a facet be a boolean as well? Tested it on the dashboard and it seems to be accepted.

Reply:
I see the same assumption was made here: https://github.com/algolia/instantsearch.js/blob/develop/src/connectors/query-rules/connectQueryRules.ts#L21

Reply:
Yes, I realized when doing the RIS implementation that they can also be booleans (with a toggle widget for instance).

See: https://github.com/algolia/react-instantsearch/pull/2258/files#diff-61853eb4221589c33badeb16d8725534R12

I'll update the types in IS.js.